example/rcnn/rcnn/symbol/symbol_resnet.py

```diff
@@ -199,7 +199,7 @@ def get_resnet_test(num_classes=config.NUM_CLASSES, num_anchors=config.NUM_ANCHO
 
     # classification
     cls_score = mx.symbol.FullyConnected(name='cls_score', data=pool1, num_hidden=num_classes)
-    cls_prob = mx.symbol.SoftmaxOutput(name='cls_prob', data=cls_score)
+    cls_prob = mx.symbol.softmax(name='cls_prob', data=cls_score)
```

Comment:
get_resnet_test() is for test case, we only have an image as input, there is no 'cls_prob_label'. Also, TestLoader doesn't provide labels. So there is a mismatch.